plugins/kubernetes/app/models/kubernetes/util.rb

```diff
@@ -20,5 +22,20 @@ def self.log(message, extra_info = {})
       @output_file.flush
       message
     end
+
+    def self.pod_watcher_symbol(cluster)
+      "cluster_pod_watcher_#{cluster.id}".to_sym
+    end
+
+    def self.start_watcher(cluster)
+      watcher_name = pod_watcher_symbol(cluster)
+      Watchers::ClusterPodWatcher.supervise as: watcher_name, args: [cluster.client]
+    end
+
+    def self.restart_watcher(cluster)
+      watcher = Celluloid::Actor[pod_watcher_symbol(cluster)]
+      watcher.terminate if watcher and watcher.alive?
+      start_watcher(cluster)
+    end
```

Comment:
This seems an odd place to put this logic.  Why not a class method in the ```Watchers::ClusterPodWatcher``` itself?